broker/nats.go

```diff
@@ -28,9 +28,10 @@ type Nats interface {
 }
 
 const (
-	prefix = "channel"
+	chansPrefix = "channels"
+
 	// SubjectAllChannels define the subject to subscribe to all channels messages
-	SubjectAllChannels = "channel.>"
+	SubjectAllChannels = "channels.>"
```

Comment:
Can you please describe this PR, the purpose and reason.

Reply:
@nmarcetic this is more or less evident. `channels.<chan_id>` is to make it consistent with the public API (which is `channels/<chan_id>`, and not `channel/<chan_id>`.

Reply:
In case I updated the description of the PR